.github/workflows/ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         env:
           discovery.type: single-node
         ports:
-          - 9200:9200
+          - 9000:9200
```

Comment:
I'm having trouble getting opensearch to run on a port that isn't 9200, so I'm trying to see if elasticsearch 7.x can accommodate a different port

Reply:
You can leave OpenSearch running on 9200, and just have the localhost mapping be on another port such as 9275. That way the OpenSearch server instance is happy but there won't be a localhost port use collission.

Reply:
Thank you! I decided to revert this since we're not adding OpenSearch to the CI at this time.